config/config.md

```diff
@@ -132,7 +132,7 @@ The following config option are provided by the OpenHIM. All of these options ha
     "orchestrationTSBufferMillis": 100
   },
   "statsd": {
-    // Rather use a statsd server for reporting metrics
+    // Report application metrics and transaction metrics to a statsd server
```

Comment:
Yeah, exactly. I figured why not. We use statsd for other things already (application metrics), may as well not delete this. The metrics calculations were the headache anyways. What do you think? Agree?